content/articles/secondary-dns-provider-easy-dns.markdown

```diff
@@ -46,6 +46,10 @@ From the *DNS settings* tab, you'll tell EasyDNS the primary name server. In thi
 
 Now add the IP address you configured at DNSimple: `50.31.225.92`
 
+<info>
+  **DNSMadeEasy** only accepts IPv4 for Secondary DNS
```

Comment:
Should this be `EasyDNS` instead of `DNSMadeEasy`?

Reply:
Good catch. Fixed at a910e49f304101172ad9cd1b5b65936370b3db20